side.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Introduction</title>
+<link rel="icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+<link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+<link rel=stylesheet href=se.css>
+<link rel=prev href=quick.html title="Quick Tour">
+<link rel=next href=install.html title=Install>
+<script src=docs.js></script>
+
+<h1>Selenium-IDE</h1>
+
+<h3>Introduction</h3>
+
+<p>The <em>Selenium-IDE(Integrated Development Environment) </em>is the tool you use to develop your
+    Selenium test cases. It’s an easy-to-use Chrome and Firefox extension and is generally the most
+    efficient way to develop test cases. It records the users actions in the browser for you, using
```

Comment:
Most efficient way? Eeek! It's great for quickly recording tests, fantastic for folks who can't code, and a nice way to bootstrap those first tests on a new extant site, but it's _far_ from being the most efficient way to develop tests.

Reply:
Can we back off of the usage of "IDE is for Test Cases" in general? It is a browser automation record and playback tool that can output Selenium code. This language makes it sound like we want people to create their test suites with it, and we should be discouraging people from doing that.

Reply:
Sure, Titus.